src/appendices/turtles/introduction.rst

```diff
@@ -5,7 +5,7 @@ One of the strengths of Python is that we can pull in ready-made features to
 make building our own programs easier. Instead of figuring out how to do all of
 the nitty-gritty details ourselves, we can use a *module* that already contains
 the code we need. Think of a module like a plug-and-play attachment for your
-phone or computer. All you need to do is connect it, and it works.
+phone or computer. All you need to do to use a module is to add a reference to your program.
```

Comment:
Instead of "...add a reference..", how about "...add `import turtle`..."

Reply:
Since this is a general explanation of what a module is, I think it would be wise to refrain from using turtles specifically. What about "All you need to do to use a module is import it."?